skcosmo/linear_model/_ridge.py

```diff
@@ -0,0 +1,247 @@
+import numpy as np
+
+from sklearn.model_selection import KFold
+from sklearn.base import RegressorMixin, MultiOutputMixin
+from sklearn.metrics import check_scoring
+
+from joblib import Parallel, delayed
+
+
+class RidgeRegression2FoldCV(MultiOutputMixin, RegressorMixin):
+    """Ridge regression with an efficient 2-fold cross validation method using the SVD solver.
```

Comment:
This could use a bit more explanation: why would I want to use this, how can I use it?

Reply:
changed to
```
    r"""Ridge regression with an efficient 2-fold cross validation method using the SVD solver.
    Minimizes the objective function:

        .. math:: \|y - Xw\|^2_2 + \alpha \|w\|^2_2, % # noqa: W605

    while the alpha value is determined with a 2-fold cross validation from a list of
    alhpa values. It is more efficient than doing a 2-fold cross validation using
    :obj:`sklearn.linear_model.RidgeCV`.
    The advantage over :obj:`sklearn.linear_model.RidgeCV` using LOOCV needs to be
    analyzed more in detail. Internal benchmarks suggest that it is more efficient
    than the LOOCV in :obj:`sklearn.linear_model.RidgeCV` for feature sizes < 600
    and in general more accurate, see issue #40. However, it is constraint to a svd
    solver for the matrix inversion.
    It offers additional functionalities in comparison to :obj:`sklearn.linear_model.Ridge`:
    The regularaization parameters can be chosen to be relative to the largest eigenvalue
    of the inverted matrix, and a cutoff regularization method is offered which is explained
    in the `Parameters` in detail.